src/Compilers/Core/Portable/Operations/BasicBlock.cs

```diff
@@ -79,12 +113,18 @@ public ImmutableArray<ControlFlowBranch> Predecessors
             }
         }
 
+        /// <summary>
+        /// Unique ordinal for each basic block in a <see cref="ControlFlowGraph"/>.
+        /// </summary>
         public int Ordinal { get; }
```

Comment:
>int [](start = 15, length = 3)

Does the int value have any meaning? Can it be zero or negative?

Reply:
It is just a unique int per basic block with no meaning. It is zero for the entry block and positive for the rest.

---
In reply to: [193190771](https://github.com/dotnet/roslyn/pull/27477#discussion_r193190771) [](ancestors = 193190771)

Reply:
> It is just a unique int per basic block with no meaning. It is zero for the entry block and positive for the rest.

It is an ordinal within the ```Blocks``` array, I think the comment should say that